conda/common/logic.py

```diff
@@ -888,12 +888,6 @@ def sum_val(sol, odict):
         return bestsol, bestval
 
 
-def evaluate_eq(eq, sol):
```

Comment:
That function is only used in the tests => this commit moves it to where it's used.